packages/xrpl/HISTORY.md

```diff
@@ -3,6 +3,8 @@
 Subscribe to [the **xrpl-announce** mailing list](https://groups.google.com/g/xrpl-announce) for release announcements. We recommend that xrpl.js (ripple-lib) users stay up-to-date with the latest stable release.
 
 ## Unreleased
+### BREAKING CHANGES
+* Small fix in the API to use `tfNoDirectRipple` instead of `tfNoRippleDirect`
```

Comment:
I think you would still want to mention it's a flag name changed for more clarity